dotCMS/src/main/webapp/WEB-INF/portlet.xml

```diff
@@ -760,6 +760,11 @@
 		<portlet-class>com.dotcms.spring.portlet.PortletController</portlet-class>
 	</portlet>
 
+	<portlet>
+		<portlet-name>content-types-angular</portlet-name>
+		<display-name>Content Types Angular</display-name>
+		<portlet-class>com.dotcms.spring.portlet.PortletController</portlet-class>
+	</portlet>
```

Comment:
Same as above, it is a 4.2 only change and should live only in master, I think something went wrong with a merge